library/hls/src/main/java/com/google/android/exoplayer2/source/hls/playlist/HlsMediaPlaylist.java

```diff
@@ -42,6 +42,11 @@
      * used for all segments that share an EXT-X-MAP tag.
      */
     @Nullable public final Segment initializationSegment;
+    /**
+     * DRM initialization data for sample decryption, or null if none of the segment uses sample
```

Comment:
Should probably be changed to

{@link DrmInitData} for sample decryption, or null if the segment's samples are not DRM-protected.